packages/node_modules/@ciscospark/internal-plugin-conversation/test/unit/spec/decrypt-transforms.js

```diff
@@ -27,6 +27,25 @@ describe('plugin-conversation', () => {
+        };
+
+        transform.fn(ctx, key, activity);
+        assert.equal(transformStub.lastCall.args[0], 'decryptPropCardItem');
```

Comment:
@taymoork2 should Kishore use an `expect` from `chai` here instead?

Reply:
Got it